flang/test/Lower/OpenMP/delayed-privatization-default-init.f90

```diff
@@ -0,0 +1,27 @@
+! Test delayed privatization for derived types with default initialization.
+
+! RUN: %flang_fc1 -emit-hlfir -fopenmp -mmlir --openmp-enable-delayed-privatization \
+! RUN:   -o - %s 2>&1 | FileCheck %s
+! RUN: bbc -emit-hlfir -fopenmp --openmp-enable-delayed-privatization -o - %s 2>&1 |\
```

Comment:
For my education, what does `bbc` tool do?

Reply:
`bbc` is an alternative frontend of the compiler, which generates the FIR (or HLFIR in this case) level. There is a `tco` second part that runs optimisation passes and generated LLVM-IR (or lowered MLIR). 

The main point of having both here is to check that the option for delayed privatization works for both variants. Almost all code is shared, but command-line parsing is not identical. [We should, as a general case, avoid running everything in both `bbc` and `flang`, if there's no clear reason why code takes a different path through the two tools].